lib/generapp/actions/develop.rb

```diff
@@ -32,14 +23,31 @@ def configure_dalli
 
         RUBY
 
-        inject_into_file(
-            'config/environments/development.rb',
-            config,
-            after: "config.action_mailer.raise_delivery_errors = true\n")
+        inject_into_file 'config/environments/development.rb',
+                         config,
+                         after: "config.action_mailer.raise_delivery_errors = true\n"
       end
 
       def configure_generators
-        config = <<-RUBY
+        inject_into_class 'config/application.rb',
+                          'Application',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.